src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.Everything/plugin.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "ID": "2140FC9819AD43A3A616E2735815C27D",
+  "ActionKeyword": "@",
+  "IsGlobal": true,
+  "Name": "Everything",
+  "Author": "IzaiahSun",
+  "Version": "1.0.0",
+  "Language": "csharp",
+  "Website": "https://aka.ms/PowerToys",
```

Comment:
Please link to your GitHub account.

Reply:
Should I replace the "website" with my GitHub home page?

Reply:
yes. with `github.com/profile` or something similar.